frontend/packages/noobaa-storage-plugin/integration-tests/tests/overviewDashboard.scenario.ts

```diff
@@ -0,0 +1,69 @@
+import { execSync } from 'child_process';
+import { browser, ExpectedConditions as until } from 'protractor';
+import { appHost, testName } from '@console/internal-integration-tests/protractor.conf';
+import {
+  obcCount,
+  clusterHealth,
+  efficiencyValue,
+  savingsValue,
+} from '../views/noobaaDashboardPage.view';
+
+const testBucket = {
+  apiVersion: 'objectbucket.io/v1alpha1',
+  kind: 'ObjectBucketClaim',
+  metadata: {
+    namespace: testName,
+    name: 'test-bucket',
+  },
+  spec: {
+    ssl: false,
+    storageClassName: 'openshift-storage.noobaa.io',
+    generateBucketName: 'test-bucket',
+  },
+};
+
+describe('Tests Object Service Dashboard Page', () => {
+  beforeAll(async () => {
```

Comment:
We need to start from this page for each of the `it` blocks. So, wouldn't this be `beforeEach` ?
I think `beforeAll` just does a one time setup at the beginning of the tests.

Reply:
If we are not changing pages do we need beforeEach?

Reply:
It's better to ensure that you are on the correct page before you start.
Also, your code already does a `get` on every `it` block, which is why I suggested a `beforeEach`.
If you think, `beforeAll` is enough, remove `get` calls from all the other `it` blocks.